pkg/cli/cli.go

```diff
@@ -429,6 +430,13 @@ func (c *cli) addSubcommands() {
 	if c.version != "" {
 		c.cmd.AddCommand(c.newVersionCmd())
 	}
+
+	alpha := &cobra.Command{
+		Use:   "alpha",
+		Short: "Alpha kubebuilder subcommands",
+	}
+	c.cmd.AddCommand(alpha)
+	alpha.AddCommand(configgen.NewCommand())
```

Comment:
Several comments about this.
1. I would create the `alpha` subcommand in its own file similar to `create`:
```
func (cli) newAlphaCmd() *cobra.Command {
	return &cobra.Command{
		Use:        "alpha",
		SuggestFor: []string{"experimental"},
		Short:      "Alpha kubebuilder commands",
		Long:       `Alpha kubebuilder commands for unstable features.
Backwards compatibility guarantee of any kind is NOT provided.`,
	}
}
```
2. Commands here are sorted in alphabetical order.
3. I would also follow the only-add-if-subcommands approach used for create.
```
	// kubebuilder alpha
	alphaCmd := c.newAlphaCmd()
	alphaCmd.AddCommand(configgen.NewCommand())
	if alphaCmd.HasSubCommands() {
		c.cmd.AddCommand(alphaCmd)
	}
```
4. I think Cobra allows us to hide some commands from help and alpha would be a very good candidate.

Reply:
> I think Cobra allows us to hide some commands from help and alpha would be a very good candidate.

Why don't we want this to be discoverable?  Is alpha insufficient?

Reply:
> I would create the alpha subcommand in its own file similar to create

Done

> Commands here are sorted in alphabetical order.

Done
> I would also follow the only-add-if-subcommands approach used for create.

Done.  I put this logic in the creation command itself so it is more encapsulated and reduces the complexity of the `cli.go` file.

Reply:
> > I think Cobra allows us to hide some commands from help and alpha would be a very good candidate.
> 
> Why don't we want this to be discoverable? Is alpha insufficient?

No strong opinion here, they can stay as they are.